pkgs/by-name/go/gow/package.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gow";
+  version = "unstable-2023-08-26";
+  rev = "af11a6e1e9ebccdcdace2a6df619355b85494d74";
+
+  src = fetchFromGitHub {
+    owner = "mitranim";
+    repo = pname;
+    inherit rev;
```

Comment:
Keeping the rev outside is redundant. If you need it elsewhere in the derivation, you can use it as `src.rev`